disperser/apiserver/server.go

```diff
@@ -450,6 +450,11 @@ func (s *DispersalServer) checkRateLimitsAndAddRatesToHeader(ctx context.Context
 		// Update the quorum rate
 		blob.RequestHeader.SecurityParams[i].QuorumRate = accountRates.Throughput
 
+		// Update AccountID to accountKey
+		// This is a combination of origin and authenticatedAddress
+		// AccountId is later used to track blobs sent by the same account
+		blob.RequestHeader.BlobAuthHeader.AccountID = accountKey
```

Comment:
This might need to change depending on if the authenticated endpoint is being used right ?